cmd/lakectl/cmd/doctor.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+// doctorCmd represents the doctor command
+var doctorCmd = &cobra.Command{
+	Use:   "doctor",
+	Short: "Run a basic diagnosis of the LakeFS configuration",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getClient()
+		rsp, err := client.ListRepositoriesWithResponse(cmd.Context(), &api.ListRepositoriesParams{})
+
+		if err != nil {
+			if strings.Contains(err.Error(), "unsupported protocol scheme") {
+				fmt.Println("It looks like you have a problem with your '.lakectl.yaml' file. \nProbably your endpoint url is wrong.")
+			} else {
+				fmt.Println(err.Error())
+			}
+		} else {
+			if rsp != nil {
+				if rsp.JSON200 != nil {
+					fmt.Println("LakeFS doctor could not find any configuration issues")
+					return
+				}
+				if rsp.JSON401 != nil {
+					fmt.Println(rsp.JSON401.Message)
+					fmt.Println("It looks like you have a problem with your '.lakectl.yaml' file. \nIt is possible that the 'access_key_id' or 'secret_access_key' you supplied are wrong.")
```

Comment:
1. Use backquoted strings, then they can be multiline _inside the same source code_.
1. Even better, `lakectl` has a template engine.
1. Even even better, if you returned some strings then you could avoid repeating "It looks like you have a problem...".
1. Also try to specify which file was used (I think; it might not even be called `.lakectl.yaml`, so best not state its name if we haven't checked it).  This too will be easier if we split the code up into a function which tries to diagnose, and reporting from the diagnosis to the user.